Add alternative instructions for ENTRYPOINT in serverless-init (#19552)
DataDog/documentation: content/en/serverless/azure_container_apps/_index.md

| `DD_SOURCE`       | See [Unified Service Tagging][5].                                       |
| `DD_TAGS`         | See [Unified Service Tagging][5].                                       |

## Alternative configurations

### Alternatives to entrypoint

> If you have already defined an entrypoint inside of your Dockerfile, you can instead modify the CMD argument.

Reviewer: ```suggestion
If you have already defined an entrypoint inside of your Dockerfile, you can modify the CMD argument instead.
```

> **Note**: As long as your command to run is passed as an argument to datadog-init, you will receive full instrumentation.

Reviewer: ```suggestion
**Note**: To receive full instrumentation, pass the command to run as an argument to `datadog-init`.
```

> The below code snippet illustates how to modify your Dockerfile to instrument Container Apps without modifying your entrypoint.

Reviewer: ```suggestion
The following code snippets show the original Dockerfile and then the modified Dockerfile to instrument Container Apps without modifying your entrypoint.
```
